src/dali/abstract_pair_converter_plugin.py

```diff
@@ -87,6 +84,10 @@ def historical_price_type(self) -> str:
     def fiat_list(self) -> List[str]:
         return self.__fiat_list
 
+    @property
```

Comment:
I don't think we need this read-only property: the issues URL is only used privately inside this class and it should not be part of the public API. You can just access __ISSUES_URL directly in this code.